src/main/kotlin/org/wfanet/measurement/duchy/DuchyPublicKeys.kt

```diff
@@ -87,20 +84,26 @@ private fun ConfigMapEntry.toDuchyPublicKeysEntry(): DuchyPublicKeys.Entry {
       "Expected $ELEMENT_SIZE bytes for element. Got ${combinedElGamalElement.size()}."
     }
     return DuchyPublicKeys.Entry(
-      publicKeyMap = elGamalElementsMap.mapValues {
-        require(it.value.size() == ELEMENT_SIZE) {
-          "Expected $ELEMENT_SIZE bytes for element. Got ${it.value.size()}."
-        }
-        ElGamalPublicKey.newBuilder().apply {
-          generator = elGamalGenerator
-          element = it.value
-        }.build()
-      },
+      publicKeyMap =
```

Comment:
This formatting is super confusing, should we pull our arguments to variables?

Reply:
Will do.

Reply:
Done.